scripts/filter_multimappers.py

```diff
@@ -128,17 +128,16 @@ def find_best_alignments(
 
     aln_indels = [(aln, count_indels(aln=aln)) for aln in alns]
     min_indels = min(aln_indels, key=lambda x: x[1])[1]
-    best_alignments = [
-            aln
-            for i, (aln, indels) in enumerate(aln_indels)
-            if indels == min_indels]
+    best_alignments = [aln
+                       for i, (aln, indels) in enumerate(aln_indels)
+                       if indels == min_indels]
```

Comment:
Was this done by `black`?

Reply:
Okay. But I would actually recommend to use 'black' and include a `black --check` call in the CI. That way, we will always have the same general style without any hassles.